### DISCUSSION TITLE
Some confusions in quickstart tutorial of self hosting

### DISCUSSION THREAD

**Discussion by otailai** [fief-dev/fief#304] [Q&A]
Here are my operation steps:

1. run command docker run -it --rm ghcr.io/fief-dev/fief:latest fief quickstart --docker,  and then, I typed my email and password, after that, I got reponse in command line as following:
`docker run \
  --name fief-server \
  -p 8000:8000 \
  -d \
  -e "SECRET=XXX" \
  -e "FIEF_CLIENT_ID=XXX" \
  -e "FIEF_CLIENT_SECRET=XXX" \
  -e "ENCRYPTION_KEY=XXX" \
  -e "PORT=8000" \
  -e "ROOT_DOMAIN=localhost:8000" \
  -e "FIEF_DOMAIN=localhost:8000" \
  -e "FIEF_MAIN_USER_EMAIL=anne@bretagne.duchy" \
  -e "FIEF_MAIN_USER_PASSWORD=XXX" \
  -e "CSRF_COOKIE_SECURE=False" \
  -e "SESSION_DATA_COOKIE_SECURE=False" \
  -e "USER_LOCALE_COOKIE_SECURE=False" \
  -e "LOGIN_SESSION_COOKIE_SECURE=False" \
  -e "SESSION_COOKIE_SECURE=False" \
  -e "FIEF_ADMIN_SESSION_COOKIE_SECURE=False" \
  ghcr.io/fief-dev/fief:latest
`
2.  I execute the docker command above, the feedback is good, the container is running:
<img width="1406" alt="image" src="https://github.com/fief-dev/fief/assets/10295363/76aa50f1-0a22-48ef-af0d-55e341756319">

Unfortunately, when I open the login page(http://localhost:8000/admin) in browser, something went run
<img width="960" alt="image" src="https://github.com/fief-dev/fief/assets/10295363/e680eefa-089f-4912-917f-9fff87afbc57">

So, I start to check the logs, found my password was too weak.
<img width="1384" alt="image" src="https://github.com/fief-dev/fief/assets/10295363/e86ba4dc-bf36-47d4-84dd-811ede6348ff">

After re-running the docker start command, I get the correct feedback.

My confusion is that maybe we can enhance the password inputing part, so our users won't debug anymore when they have such problems. 

Thank you.

**Comment by fief-bailiff** [comment]
Hail, @otailai 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Good catch! We could indeed proactively verify the password in the CLI! I'll open an issue for that.

**Reply by frankie567** [reply]
Tracked in #305
